addons/l10n_ar/views/account_move_view.xml

```diff
@@ -22,7 +22,7 @@
         <field name="inherit_id" ref="account.view_move_form"/>
         <field name="arch" type="xml">
             <field name="fiscal_position_id" position="before">
-                <field name="l10n_ar_afip_responsibility_type_id" options="{'no_open': True, 'no_create': True}" attrs="{'invisible': ['|', ('state', 'not in', ['open', 'paid']), ('l10n_latam_country_code', '!=', 'AR')]}"/>
+                <field name="l10n_ar_afip_responsibility_type_id" options="{'no_open': True, 'no_create': True}" attrs="{'invisible': ['|', ('state', 'not in', ['posted']), ('l10n_latam_country_code', '!=', 'AR')]}"/>
```

Comment:
I don't understand why this field responsibility type is here in the first place.  It should be the one of the partners and even if it is posted, this should not be visible.  I can understand there is a case where the responsibility type changes, but this is not simplifying things.

Reply:
@jjscarafia  thank you, I change it to the way you suggested.

-- 
@jco-odoo 

The reason why the AFIP Responsibility field exist in the account.move model is because we are letting to change the AFIP Responsibility in the partner on any moment. 

An AFIP Responsibility could change in a partner because the partner has a new categorization on AFIP, or maybe because the responsibility was wrongly configure in the first place and then is changed it to set the proper one.

In any of this cases if the AFIP responsibility change we are able to track the AFIP responsibility of the partner when an invoice has been validated (which affect in the selected document type, the VAT taxes on the invoice line, etc).

Reply:
@zaoral But on the other hand, we will also be keeping all information about what we sent/received in the database.  This is overkill and only confusing.  The field is there, so we can keep it, but I would simply remove it from the view.  A better solution for this might be to put a tracking on the afip responsibility type in the chatter of the partner.

Reply:
Hi @jco-odoo , I think that probably we could remove it from the view @jjscarafia Do you agree?

Reply:
yes, no problem, we can remove it from the view